tests/acceptance/test_login.py

```diff
@@ -35,4 +35,4 @@ def test_login(selenium, small_app):
     email.send_keys('admin@inspirehep.net')
     password = selenium.find_element_by_id('password')
     password.send_keys('123456')
-
+    selenium.find_element_by_xpath("//button[@type='submit']").click()
```

Comment:
Could be committed separately but at least is adding the final bit that was missing to mine. Maybe would also be nice to test that the login was successful